### DISCUSSION TITLE
Sometime category images didn't load

### DISCUSSION THREAD

**Discussion by webcreative24** [breezefront/community#169] [1. Help]
Hello @vovayatsyuk . We are stuck in some strange behavior on the category page with Breeze enabled. Sometimes I'm getting 503 errors on category images:
![image](https://github.com/breezefront/community/assets/13699502/d6814ef1-d880-4bd2-bbc7-90c0c63afc5c)
![image](https://github.com/breezefront/community/assets/13699502/82abb67f-0881-4d54-9e18-dbcb212f6ba6)
But when I copy the link for the broken image and open this image directly in the browser, everything is loading and after refreshing the category page it's loading too but for the first visit, it can be an issue with the image load for some images

Magento 2.4.5, Breeze 2.5.5
Varnish and Cloudflare enabled

Can you please help? Thanks in advance

**Comment by vovayatsyuk** [comment]
503 is a server error, maybe a RAM issue? I suggest to inspect server logs.

**Reply by webcreative24** [reply]
yes, it's a Varnish issue. Thanks for the reply
